tests/vcs/git/GitCommitBearTest.py

```diff
@@ -109,17 +109,17 @@ def test_pure_oneliner_message(self, patch):
         self.assertTrue(self.msg_queue.empty())
 
     def test_shortlog_checks_length(self):
-        self.git_commit("Commits messages that nearly exceeds default limit")
+        self.git_commit("Commit messages that nearly exceed default limit")
```

Comment:
please don't change this one, though it's wrong english the number of chars in it is on purpose, as (like it tries to state) uses up 50 chars which nearly exceeds the limit.
If you want put a comment above that^^